src/BlockchainWriter/BlockchainWriter.ts

```diff
@@ -44,28 +42,63 @@ export class BlockchainWriter {
     this.mongoClient = await MongoClient.connect(this.configuration.dbUrl)
     const db = await this.mongoClient.db()
 
-    const blockchainWriterCollection = db.collection('blockchainWriter')
-    const blockchainInfoCollection = db.collection('blockchainInfo')
+    const blockchainWriterCollection: Collection = db.collection('blockchainWriter')
+    const blockchainInfoCollection: Collection = db.collection('blockchainInfo')
```

Comment:
This is not used anywhere. From looking at the previous code, this collection and the dao for it don't seem to be used at all. I am going to create a new issue to delete these.

Reply:
Yes this was an earlier idea you and I had discussed. I'll remove it